### PR TITLE
CI requires cmake>=3.18 to install PyTorch

### PR DESCRIPTION
**Pull request by yeounoh**
PyTorch just updated their `cmake` requirement.

### CODE REVIEW

**Comment by JackCaoG** [comment]
Do we still need this pr?

**Comment by yeounoh** [comment]
> Do we still need this pr?

I think so, to keep in pace with PyTorch. This doesn't break our end -- let me rebase and make this PR green, and then we can merge.

File: torch_xla/csrc/aten_xla_type.cpp [hunk 1]
```diff
@@ -1105,8 +1105,8 @@ at::Tensor XLANativeFunctions::embedding_symint(const at::Tensor& weight,
   XLA_FN_COUNTER("xla::");
   // TODO: for now route to native, which dispatches supported XLA operations.
   // We need to make use of the TPU embedding core here eventually.
-  return at::native::embedding_symint(weight, indices, padding_idx,
-                                      scale_grad_by_freq, sparse);
+  return at::embedding(weight, indices, padding_idx, scale_grad_by_freq,
```
[the remark this answers is not in the document]
Comment (yeounoh) [reply torch_xla/csrc/aten_xla_type.cpp]: It was actually a different issue -- some upsteram API change. Rebased, and this change is not needed.

**Review by JackCaoG — APPROVED** [review]
fell free to merge once CI is green.

**Pull request merged**